frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js

```diff
@@ -510,15 +510,16 @@ const SubmissionsCreate = ({ path }) => {
     stateSla = null,
     isAssignedRole = false,
     assignedRole = [],
+    assignees,
   }) => {
-    const assignes = !isAssignedRole ? [userInfo?.uuid] || [] : [];
+    const assignes = assignees ? assignees : !isAssignedRole ? [userInfo?.uuid] || [] : [];
     await submissionService.customApiService(Urls.application.pendingTask, {
       pendingTask: {
         name,
         entityType,
         referenceId: `MANUAL_${refId}`,
         status,
-        assignedTo: assignes?.map((uuid) => ({ uuid })),
+        assignedTo: assignees ? assignes : assignes?.map((uuid) => ({ uuid })),
```

Comment:
Remove the condition from here. Declare const assignees  as const finalAssigness and map those here.
finalAssignees should always have assignees in it

Reply:
done